packages/rum-core/src/domain/rumEventsCollection/action/trackActions.spec.ts

```diff
@@ -234,60 +190,51 @@ describe('newAction', () => {
       resetExperimentalFeatures()
     })
 
-    it('collect actions even if another one is ongoing', () => {
+    it("doesn't discard pending action on view created", () => {
       const { lifeCycle, domMutationObservable, clock } = setupBuilder.build()
-      lifeCycle.subscribe(LifeCycleEventType.AUTO_ACTION_COMPLETED, pushEvent)
+      mockValidatedClickAction(domMutationObservable, clock)
+      expect(findActionId()).not.toBeUndefined()
+
+      lifeCycle.notify(LifeCycleEventType.VIEW_CREATED, {
+        id: 'fake',
+        startClocks: jasmine.any(Object) as unknown as ClocksState,
+      })
+      clock.tick(EXPIRE_DELAY)
+
+      expect(events.length).toBe(1)
+    })
 
-      newClick('test-1')
-      newClick('test-2')
+    it('collect actions even if another one is ongoing', () => {
+      const { domMutationObservable, clock } = setupBuilder.build()
 
-      clock.tick(BEFORE_PAGE_ACTIVITY_VALIDATION_DELAY)
-      domMutationObservable.notify()
+      const firstClickTimeStamp = timeStampNow()
+      mockValidatedClickAction(domMutationObservable, clock)
+      const secondClickTimeStamp = timeStampNow()
+      mockValidatedClickAction(domMutationObservable, clock)
 
       clock.tick(EXPIRE_DELAY)
       expect(events.length).toBe(2)
-      expect(events[0].name).toBe('test-1')
-      expect(events[1].name).toBe('test-2')
+      expect(events[0].startClocks.timeStamp).toBe(firstClickTimeStamp)
+      expect(events[1].startClocks.timeStamp).toBe(secondClickTimeStamp)
     })
   })
 
-  it('counts errors occurring during the action', () => {
-    const { lifeCycle, domMutationObservable, clock } = setupBuilder.build()
-    const collectedRumEvent = { type: RumEventType.ERROR } as RumEvent & Context
-    lifeCycle.subscribe(LifeCycleEventType.AUTO_ACTION_COMPLETED, pushEvent)
-
-    newClick('test-1')
-
-    lifeCycle.notify(LifeCycleEventType.RUM_EVENT_COLLECTED, collectedRumEvent)
-    clock.tick(BEFORE_PAGE_ACTIVITY_VALIDATION_DELAY)
-    domMutationObservable.notify()
-    lifeCycle.notify(LifeCycleEventType.RUM_EVENT_COLLECTED, collectedRumEvent)
+  it('discards a action when nothing happens after a click', () => {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** not related to your PR but ‏several occurrences of `a action` could be corrected to `an action`